benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -279,9 +280,9 @@ def get_all_optimizer_operators():
 
     # Filter for Optimizer operators
     optimizer_mx_operators = {}
-    for op_name, _ in mx_operators.items():
-        if op_name in optimizer_ops and op_name not in unique_ops:
-            optimizer_mx_operators[op_name] = mx_operators[op_name]
+    for op_name, op_params in mx_operators.items():
+         if op_name in optimizer_ops:
```

Comment:
wait doesn't this give indent error?

Reply:
Didn't throw an error, but fixed regardless.